test/integration/protocol_integration_test.cc

```diff
@@ -1190,6 +1192,85 @@ TEST_P(DownstreamProtocolIntegrationTest, MultipleContentLengths) {
   }
 }
 
+// override_stream_error_on_invalid_http_message=true and HCM
+// stream_error_on_invalid_http_message=false: test that HTTP/1.1 connection is left open on invalid
+// HTTP message (missing :host header)
+TEST_P(DownstreamProtocolIntegrationTest,
+       Http1ConnectionIsLeftOpenIfHCMStreamErrorIsFalseAndOverrideIsTrue) {
+  if (downstreamProtocol() == Http::CodecClient::Type::HTTP2) {
```

Comment:
I think you can move these to 
TEST_P(IntegrationTest,...)
and then it won't kick off a test run for downstream HTTP/2